portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1199,8 +1199,8 @@ definition {
 
 	macro viewOptionSingleSelection {
 		AssertElementPresent(
-			key_optionName = "${optionName}",
 			key_fieldName = "${fieldName}",
+			key_optionName = "${optionName}",
```

Comment:
Almost there, @gleicelisbino, we just need to perform this same sort on the calls of this macro too. :wink:

Reply:
@natocesarrego  :0

Reply:
Sorry. :(